src/simulator/handlers.js

```diff
@@ -94,6 +91,21 @@ const postTransfers = async (ctx) => {
     }
 };
 
+const putTransfersById = async (ctx) => {
+    try {
+        const res = await ctx.state.model.transfer.update(ctx.state.path.params.transferId, {
+            ...ctx.request.body,
+            homeTransactionId: ctx.state.path.params.transferId,
```

Comment:
@vgenev I think homeTransactionId should be null here for an incoming transfer i.e. when the simulator is acting as payee. The home transaction Id is not known here and is definitely not the mojaloop transferId.